asn1ate/support/pygen.py

```diff
@@ -105,3 +105,22 @@ def __init__(self, indent_size=4):
 
     def __str__(self):
         return self.buf.getvalue()
+
+
+def format_longstring(content):
+    """ Wrap content in a Python longstring format.
+
+    E.g. format_longstring("Hello") will return:
+
+    \"\"\"
+    Hello
+    \"\"\"
+
+    Inline quote chars are escaped.
+    """
+    lines = [
+        '"""',
+        content.replace('"', '\\"'),
```

Comment:
It might be good to also replace `\` with a double backslash, especially if a `\` can occur in the ASN.1.

Reply:
I completely lost track of this; spring is usually my most busy time of the year. But now, a year later, I had some spare time and rediscovered the branch. :)

You're right -- we want to escape both `\` and `"""`, but `\"` itself is not necessary.

I found a number of bugs in the Python code-generation for ASN.1 with exotic escape sequences, but this source embedding should work OK now.